plugin/src/main/groovy/com/btkelly/gnag/reporters/BaseViolationDetector.groovy

```diff
@@ -27,58 +25,4 @@ abstract class BaseViolationDetector implements ViolationDetector {
         this.project = project
     }
 
-    /**
-     * @param absoluteFilePath the absolute path to a target file. Should be trimmed of all excess whitespace and
-     *                         newlines. Can be null.
-     * @return the path to the target file, relative to the root directory of the current project, iff the target file
-     *         is contained within that directory; null otherwise
-     */
-    protected String computeFilePathRelativeToProjectRoot(final String absoluteFilePath) {
```

Comment:
Moved to allow Checkstyle parsing code (now shared by Checkstyle and ktlint checkers) to call these methods from outside the `reporters` package.